content/modules/redisgears/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+Title: RedisGears
+description:
+weight: 55
+alwaysopen: false
+categories: ["Modules"]
+---
+## What is RedisGears?
+
+RedisGears is an engine for data processing in Redis. RedisGears supports batch and event-driven processing for Redis data. To use RedisGears, you write functions that describe how your data should be processed. You then submit this code to your Redis deployment for remote execution.
+
+As of v1.0.0, code for RedisGears must be written in Python. However, an internal C API exists and can be used by other Redis modules. In addition, support for other languages, including Java and JavaScript, is being planned.
+
+## Getting Started with RedisGears
+
+RedisGears is implemented by a Redis module. To use RedisGears, you'll need to make sure that your Redis deployment has the module installed. [Redis Cloud](https://redislabs.com/redis-enterprise-cloud/) and [Redis Enterprise](https://docs.redislabs.com/latest/rs/) both support the module natively.
+
+If you're running open source Redis, you'll also need to [install the module](https://oss.redislabs.com/redisgears/quickstart.html) before using it.
```

Comment:
@bmansheim this is a good way to point to it in the RedisAI docs in stead of docker